extractor/src/main/scala/com/digitalasset/extractor/Extractor.scala

```diff
@@ -42,6 +46,7 @@ class Extractor[T](config: ExtractorConfig, target: T)(
     extends StrictLogging {
 
   private val tokenHolder = config.accessTokenFile.map(new TokenHolder(_))
+  private val parties = config.parties.toSet.map[String, Set[String]](identity)
```

Comment:
```suggestion
  private val parties = config.parties.widen[String].toSet
```

Reply:
Yay, wartremover:
```
[wartremover:Any] Inferred type containing Any
  private val parties: List[String]= config.parties.widen[String].toList
```

Reply:
I have no idea where the `Any` is coming from.

Reply:
My best guess is `ToFunctorOps[({type l[a >: Nothing <: Any] = Set[a]})#l, String]`. wartremover/wartremover#463

Reply:
Thanks.